cachi2/core/models/sbom.py

```diff
@@ -104,3 +110,361 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self) -> "SPDXSbom":
```

Comment:
In merge SPDX tests I get `['SPDXRef-DocumentRoot-File-', None]` for envelopes, this piece of code makes me think that this could be a bug. What behavior is correct?